### ISSUE TITLE
baritone bot go to random location and path finding forever

### ISSUE THREAD

**Issue by mr-noob-roblox** [cabaletta/baritone#3046]
## Some information
Operating system: windows 10
Java version: 8
Minecraft version: 1.16
Baritone version: idk i install it from impact

## Exception, error or logs
when the bot build schematic some times it stop and forever pathfinding and also go to random location to do the forever path finding

## How to reproduce
just build schematic and wait

## Modified settings
*alot and dont have time to write it*

**Comment by scorbett123** [comment]
Can you send a log? Also what other mods are you using if any?

**Comment by scorbett123** [comment]
ok,  without a log we can't do anything, can you try to reproduce this issue and then send the log

**Comment by HackerPhat** [comment]
> ok, without a log we can't do anything, can you try to reproduce this issue and then send the log

thought it will took a while to trigger the bug but at the start it just build a bit then go to my old build the one that trigger the bug and it path finding forever
also here is the log:
[07:30:42] [main/INFO]: Using primary tweak class name clientapi.load.ClientTweaker
[07:30:42] [main/INFO]: Loading tweak class name baritone.launch.BaritoneTweaker
[07:30:42] [main/INFO]: Calling tweak class clientapi.load.ClientTweaker
[07:30:42] [main/INFO]: Injecting into ClassLoader
[07:30:42] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.7.11 Source=file:/C:/Users/40C%20LTN/AppData/Roaming/.minecraft/libraries/org/spongepowered/mixin/0.7.11-SNAPSHOT/mixin-0.7.11-SNAPSHOT.jar Service=LaunchWrapper Env=UNKNOWN
[07:30:43] [main/INFO]: FML platform manager could not load class cpw.mods.fml.relauncher.CoreModManager. Proceeding without FML support.
[07:30:43] [main/INFO]: Initialized Mixin bootstrap
[07:30:43] [main/INFO]: Setup Mixin Environment
[07:30:43] [main/INFO]: Compatibility level set to JAVA_8
[07:30:43] [main/ERROR]: Mixin config mixins.impact.json does not specify "minVersion" property
[07:30:43] [main/INFO]: Loaded Mixin Configurations
[07:30:43] [main/INFO]: Calling tweak class baritone.launch.BaritoneTweaker
[07:30:43] [main/ERROR]: Mixin config mixins.baritone.json does not specify "minVersion" property
[07:30:43] [main/INFO]: Loading tweak class name org.spongepowered.asm.mixin.EnvironmentStateTweaker
[07:30:43] [main/INFO]: Calling tweak class org.spongepowered.asm.mixin.EnvironmentStateTweaker
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinBipedArmorLayer, name: clientapi.load.mixin.MixinBipedArmorLayer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinBlockMode, name: clientapi.load.mixin.MixinBlockMode
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinBossOverlayGui, name: clientapi.load.mixin.MixinBossOverlayGui
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinClientPlayerEntity, name: clientapi.load.mixin.MixinClientPlayerEntity
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinClientPlayNetHandler, name: clientapi.load.mixin.MixinClientPlayNetHandler
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinEntity, name: clientapi.load.mixin.MixinEntity
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinEntityRenderer, name: clientapi.load.mixin.MixinEntityRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinEntityRendererManager, name: clientapi.load.mixin.MixinEntityRendererManager
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinFirstPersonRenderer, name: clientapi.load.mixin.MixinFirstPersonRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinFogRenderer, name: clientapi.load.mixin.MixinFogRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinFontRenderer, name: clientapi.load.mixin.MixinFontRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinGameRenderer, name: clientapi.load.mixin.MixinGameRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinIngameGui, name: clientapi.load.mixin.MixinIngameGui
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinItemRenderer, name: clientapi.load.mixin.MixinItemRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinKeyboardListener, name: clientapi.load.mixin.MixinKeyboardListener
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinLivingEntity, name: clientapi.load.mixin.MixinLivingEntity
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinMinecraft, name: clientapi.load.mixin.MixinMinecraft
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinMouseHelper, name: clientapi.load.mixin.MixinMouseHelper
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinNettyPacketDecoder, name: clientapi.load.mixin.MixinNettyPacketDecoder
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinNettyPacketEncoder, name: clientapi.load.mixin.MixinNettyPacketEncoder
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinNetworkManager, name: clientapi.load.mixin.MixinNetworkManager
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinOverlayRenderer, name: clientapi.load.mixin.MixinOverlayRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinParticleManager, name: clientapi.load.mixin.MixinParticleManager
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinRenderLivingBase, name: clientapi.load.mixin.MixinRenderLivingBase
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinSpectatorGui, name: clientapi.load.mixin.MixinSpectatorGui
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinTimer, name: clientapi.load.mixin.MixinTimer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.capi.json:MixinWorldRenderer, name: clientapi.load.mixin.MixinWorldRenderer
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinChunkArray, name: baritone.launch.mixins.MixinChunkArray
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinClientChunkProvider, name: baritone.launch.mixins.MixinClientChunkProvider
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinClientPlayerEntity, name: baritone.launch.mixins.MixinClientPlayerEntity
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinClientPlayNetHandler, name: baritone.launch.mixins.MixinClientPlayNetHandler
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinCommandSuggestionHelper, name: baritone.launch.mixins.MixinCommandSuggestionHelper
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinEntity, name: baritone.launch.mixins.MixinEntity
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinEntityRenderManager, name: baritone.launch.mixins.MixinEntityRenderManager
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinItemStack, name: baritone.launch.mixins.MixinItemStack
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinLivingEntity, name: baritone.launch.mixins.MixinLivingEntity
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinLootContext, name: baritone.launch.mixins.MixinLootContext
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinMinecraft, name: baritone.launch.mixins.MixinMinecraft
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinNetworkManager, name: baritone.launch.mixins.MixinNetworkManager
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinPlayerController, name: baritone.launch.mixins.MixinPlayerController
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinScreen, name: baritone.launch.mixins.MixinScreen
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinSodiumChunkProvider, name: baritone.launch.mixins.MixinSodiumChunkProvider
[07:30:43] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/world/SodiumChunkManager (java.lang.ClassNotFoundException: The specified class 'me.jellysquid.mods.sodium.client.world.SodiumChunkManager' was not found)
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinSodiumFixedLongHashTable, name: baritone.launch.mixins.MixinSodiumFixedLongHashTable
[07:30:43] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/util/collections/FixedLongHashTable (java.lang.ClassNotFoundException: The specified class 'me.jellysquid.mods.sodium.client.util.collections.FixedLongHashTable' was not found)
[07:30:43] [main/ERROR]: Classloader restrictions [PACKAGE_CLASSLOADER_EXCLUSION] encountered loading mixins.baritone.json:MixinWorldRenderer, name: baritone.launch.mixins.MixinWorldRenderer
[07:30:43] [main/INFO]: Launching wrapped minecraft {net.minecraft.client.main.Main}
[07:30:44] [main/WARN]: Injection warning: LVT in aqa::g(Ldcn;)Ldcn; has incompatible changes at opcode 148 in callback aqa::handler$getAllowedMovement$development$zcc000.
Expected: [Ldci;, Ldcs;, Lddh;, Ljava/util/stream/Stream;, Ljava/util/stream/Stream;, Lafo;, Ldcn;, Z, Z, Z, Z]
   Found: [Ldci;, Ldcs;, Ljava/util/stream/Stream;, Ljava/util/stream/Stream;, Lafo;, Ldcn;, Z, Z, Z, Z]
[07:30:55] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[07:30:57] [main/ERROR]: Failed to verify authentication
com.mojang.authlib.exceptions.AuthenticationUnavailableException: null
	at com.mojang.authlib.yggdrasil.YggdrasilSocialInteractionsService.checkPrivileges(YggdrasilSocialInteractionsService.java:97) ~[YggdrasilSocialInteractionsService.class:?]
	at com.mojang.authlib.yggdrasil.YggdrasilSocialInteractionsService.<init>(YggdrasilSocialInteractionsService.java:40) ~[YggdrasilSocialInteractionsService.class:?]
	at com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService.createSocialInteractionsService(YggdrasilAuthenticationService.java:152) ~[YggdrasilAuthenticationService.class:?]
	at djz.a(SourceFile:625) [djz.class:?]
	at djz.<init>(SourceFile:409) [djz.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:177) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
[07:30:57] [main/INFO]: Setting user: insert_a_very_epic_name
[07:30:58] [main/WARN]: Skipping bad option: textBackgroundOpacity:
[07:30:58] [main/INFO]: Backend library: LWJGL version 3.2.2 build 10
[07:31:07] [main/INFO]: Narrator library for x64 successfully loaded
[07:31:08] [main/INFO]: Reloading ResourceManager: Default, herry-pooter-e990
[07:31:10] [main/INFO]: Initializing Fonts
[07:31:14] [Realms Notification Availability checker #1/INFO]: Could not authorize you against Realms server: Invalid session id
[07:31:18] [main/INFO]: [STDERR]: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[07:31:18] [main/INFO]: [STDERR]: SLF4J: Defaulting to no-operation (NOP) logger implementation
[07:31:18] [main/INFO]: [STDERR]: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[07:31:19] [main/INFO]: [STDOUT]: Sending event category=client action=launch label=4.9.1-1.16.5 cid=1748156610.1634612245
[07:31:19] [pool-5-thread-1/INFO]: Checking for Updates
[07:31:19] [pool-5-thread-1/INFO]: [STDOUT]: Offering updates to betas: false
[07:31:19] [pool-5-thread-1/INFO]: [STDOUT]: Fetching releases from ImpactDevelopment/ImpactReleases
[07:31:19] [pool-5-thread-1/INFO]: [STDOUT]: DOWNLOADING http://impactclient.net/releases.json
[07:31:25] [main/INFO]: OpenGL debug message, id=0, source=SHADER COMPILER, type=ERROR, severity=HIGH, message=SHADER_ID_COMPILE error has been generated. GLSL compile failed for shader 7, "": ERROR: 0:3: 'varying out' : syntax error syntax error


[07:31:25] [main/INFO]: [STDERR]: net.impactclient.6m: ERROR: 0:3: 'varying out' : syntax error syntax error


[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.9A.5(9A.java:13)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8c.4(8c.java:7)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8w.7(8w.java:4)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8v.<init>(8v.java:11)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8_.<init>(8_.java:13)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8_.<init>(8_.java:7)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.7V.<init>(7V.java:90)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.2C.3(2C.java:37)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.5z.6(5z.java:74)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.3A.6(3A.java:2)
[07:31:25] [main/INFO]: [STDERR]: 	at djz.handler$init$zzp000(SourceFile:2861)
[07:31:25] [main/INFO]: [STDERR]: 	at djz.<init>(SourceFile:592)
[07:31:25] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(SourceFile:177)
[07:31:25] [main/INFO]: [STDERR]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[07:31:25] [main/INFO]: [STDERR]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[07:31:25] [main/INFO]: [STDERR]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[07:31:25] [main/INFO]: [STDERR]: 	at java.lang.reflect.Method.invoke(Method.java:497)
[07:31:25] [main/INFO]: [STDERR]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[07:31:25] [main/INFO]: [STDERR]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[07:31:25] [main/INFO]: [STDERR]: net.impactclient.6m: Cannot attach a shader to program if shader isn't generated!
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8L.0(8L.java:25)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8L$$Lambda$3073/1211046756.accept(Unknown Source)
[07:31:25] [main/INFO]: [STDERR]: 	at java.util.ArrayList.forEach(ArrayList.java:1249)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8L.4(8L.java:49)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8w.7(8w.java:4)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.8_.<init>(8_.java:3)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.7V.<init>(7V.java:90)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.2C.3(2C.java:37)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.5z.6(5z.java:74)
[07:31:25] [main/INFO]: [STDERR]: 	at net.impactclient.3A.6(3A.java:2)
[07:31:25] [main/INFO]: [STDERR]: 	at djz.handler$init$zzp000(SourceFile:2861)
[07:31:25] [main/INFO]: [STDERR]: 	at djz.<init>(SourceFile:592)
[07:31:25] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(SourceFile:177)
[07:31:25] [main/INFO]: [STDERR]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[07:31:25] [main/INFO]: [STDERR]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[07:31:25] [main/INFO]: [STDERR]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[07:31:25] [main/INFO]: [STDERR]: 	at java.lang.reflect.Method.invoke(Method.java:497)
[07:31:25] [main/INFO]: [STDERR]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[07:31:25] [main/INFO]: [STDERR]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[07:31:25] [main/INFO]: Initialized 119 mods!
[07:31:25] [main/ERROR]: ########## GL ERROR ##########
[07:31:25] [main/ERROR]: @ Post startup
[07:31:25] [main/ERROR]: 65539: Invalid key -1
[07:31:26] [main/FATAL]: Unable to load spam config file: Spam config file is in invalid format
[07:31:26] [main/INFO]: [STDOUT]: Initializing took 18194ms
[07:31:26] [main/ERROR]: Setting skin to impaste
[07:31:32] [main/INFO]: OpenAL initialized.
[07:31:32] [main/INFO]: Sound engine started
[07:31:33] [main/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/blocks.png-atlas
[07:31:33] [main/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[07:31:33] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[07:31:33] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[07:31:33] [main/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[07:31:33] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[07:31:33] [main/INFO]: Created: 2048x2048x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[07:31:37] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[07:31:37] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[07:31:37] [main/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[07:31:38] [main/ERROR]: Setting skin to impaste
[07:32:02] [main/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[07:32:02] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[07:32:02] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[07:32:02] [main/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[07:32:02] [main/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[07:32:02] [main/INFO]: Reloading ResourceManager: Default
[07:32:03] [main/INFO]: Loaded 7 recipes
[07:32:04] [main/INFO]: Loaded 927 advancements
[07:32:05] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[07:32:05] [Server thread/INFO]: Starting integrated minecraft server version 1.16.5
[07:32:05] [Server thread/INFO]: Generating keypair
[07:32:06] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[07:32:09] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:09] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:09] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:09] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:09] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:09] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:10] [main/INFO]: Preepayrin spawn ayyrea: 0%
[07:32:10] [main/INFO]: Preepayrin spawn ayyrea: 85%
[07:32:10] [main/INFO]: Preepayrin spawn ayyrea: 85%
[07:32:11] [main/INFO]: Preepayrin spawn ayyrea: 85%
[07:32:12] [main/INFO]: Preepayrin spawn ayyrea: 94%
[07:32:12] [main/INFO]: Time elapsed: 5278 ms
[07:32:12] [Server thread/INFO]: Changing view distance to 4, from 10
[07:32:14] [Server thread/INFO]: insert_a_very_epic_name[local:E:d28331b9] logged in with entity id 31 at (77.47626088724891, -1937.67960320504, 94.48613410180397)
[07:32:14] [Server thread/INFO]: insert_a_very_epic_name joind teh gaem
[07:32:14] [main/INFO]: [STDOUT]: Baritone world data dir: C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone
[07:32:14] [main/INFO]: [STDOUT]: Cached world directory: C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:14] [main/INFO]: [STDOUT]: Would save waypoints to C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\waypoints
[07:32:14] [Server thread/INFO]: Saving and pausing game...
[07:32:14] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:overworld
[07:32:14] [pool-4-thread-1/INFO]: [STDOUT]: Loading region -1,0 from disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:14] [main/INFO]: Loaded 36 advancements
[07:32:14] [pool-4-thread-1/INFO]: [STDOUT]: Loaded region successfully in 138ms
[07:32:14] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_nether
[07:32:14] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_end
[07:32:15] [pool-4-thread-1/INFO]: [STDOUT]: Loading region 0,0 from disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:15] [pool-4-thread-1/INFO]: [STDOUT]: Loaded region successfully in 307ms
[07:32:19] [pool-4-thread-1/INFO]: [STDOUT]: Loading region -1,-1 from disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:19] [pool-4-thread-1/INFO]: [STDOUT]: Loaded region successfully in 5ms
[07:32:19] [pool-4-thread-1/INFO]: [STDOUT]: Loading region 0,-1 from disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:19] [pool-4-thread-1/INFO]: [STDOUT]: Loaded region successfully in 16ms
[07:32:44] [pool-4-thread-2/INFO]: [STDOUT]: Saving region -1,0 to disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:44] [ForkJoinPool.commonPool-worker-3/INFO]: [STDOUT]: Saving region 0,0 to disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:44] [ForkJoinPool.commonPool-worker-2/INFO]: [STDOUT]: Saving region 0,-1 to disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:44] [ForkJoinPool.commonPool-worker-1/INFO]: [STDOUT]: Saving region -1,-1 to disk C:\Users\40C LTN\AppData\Roaming\.minecraft\saves\New Wurld (3)\baritone\cache
[07:32:44] [ForkJoinPool.commonPool-worker-1/INFO]: [STDOUT]: Saved region successfully
[07:32:44] [ForkJoinPool.commonPool-worker-2/INFO]: [STDOUT]: Saved region successfully
[07:32:44] [pool-4-thread-2/INFO]: [STDOUT]: Saved region successfully
[07:32:44] [ForkJoinPool.commonPool-worker-3/INFO]: [STDOUT]: Saved region successfully
[07:32:44] [pool-4-thread-2/INFO]: [STDOUT]: World save took 107ms
[07:33:20] [main/INFO]: [CHAT] [Baritone] > build redeemer -40 4 78
[07:33:20] [main/INFO]: [CHAT] [Baritone] Successfully loaded schematic for building
[07:33:20] [main/INFO]: [CHAT] [Baritone] Origin: BetterBlockPos{x=-40,y=4,z=78}
[07:33:20] [main/INFO]: [CHAT] [Baritone] Starting layer 0
[07:33:30] [Server thread/INFO]: Saving and pausing game...
[07:33:30] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:overworld
[07:33:31] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_nether
[07:33:31] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_end
[07:33:44] [pool-4-thread-4/INFO]: [STDOUT]: 2306304 movements considered
[07:33:44] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 20127
[07:33:44] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 124959
[07:33:44] [pool-4-thread-4/INFO]: [STDOUT]: 52311 nodes per second
[07:33:46] [pool-4-thread-4/INFO]: [STDOUT]: 2997632 movements considered
[07:33:46] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 24020
[07:33:46] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 160276
[07:33:46] [pool-4-thread-4/INFO]: [STDOUT]: 68093 nodes per second
[07:33:48] [pool-4-thread-4/INFO]: [STDOUT]: 3014528 movements considered
[07:33:48] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 24117
[07:33:48] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 161141
[07:33:48] [pool-4-thread-4/INFO]: [STDOUT]: 68205 nodes per second
[07:33:50] [pool-4-thread-4/INFO]: [STDOUT]: 2948352 movements considered
[07:33:50] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 23761
[07:33:50] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 157777
[07:33:50] [pool-4-thread-4/INFO]: [STDOUT]: 66974 nodes per second
[07:33:52] [pool-4-thread-4/INFO]: [STDOUT]: 3793152 movements considered
[07:33:52] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 28359
[07:33:52] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 200775
[07:33:52] [pool-4-thread-4/INFO]: [STDOUT]: 86208 nodes per second
[07:33:54] [pool-4-thread-4/INFO]: [STDOUT]: 3442560 movements considered
[07:33:54] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 26422
[07:33:54] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 182902
[07:33:54] [pool-4-thread-4/INFO]: [STDOUT]: 78161 nodes per second
[07:33:56] [pool-4-thread-4/INFO]: [STDOUT]: 3863552 movements considered
[07:33:56] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 28688
[07:33:56] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 204304
[07:33:56] [pool-4-thread-4/INFO]: [STDOUT]: 87764 nodes per second
[07:33:57] [Server thread/INFO]: Saving and pausing game...
[07:33:57] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:overworld
[07:33:57] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_nether
[07:33:57] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_end
[07:33:58] [pool-4-thread-4/INFO]: [STDOUT]: 3601664 movements considered
[07:33:58] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 27306
[07:33:58] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 191018
[07:33:58] [pool-4-thread-4/INFO]: [STDOUT]: 81856 nodes per second
[07:34:00] [pool-4-thread-4/INFO]: [STDOUT]: 2990592 movements considered
[07:34:00] [pool-4-thread-4/INFO]: [STDOUT]: Open set size: 24003
[07:34:00] [pool-4-thread-4/INFO]: [STDOUT]: PathNode map size: 159939
[07:34:00] [pool-4-thread-4/INFO]: [STDOUT]: 67968 nodes per second
[07:34:01] [Server thread/INFO]: insert_a_very_epic_name lost connection: CYA
[07:34:01] [Server thread/INFO]: insert_a_very_epic_name leeft teh gaem
[07:34:01] [Server thread/INFO]: Stopping singleplayer server as player logged out
[07:34:01] [Server thread/INFO]: Stopping server
[07:34:01] [Server thread/INFO]: Saving players
[07:34:01] [Server thread/INFO]: Saving worlds
[07:34:01] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:overworld
[07:34:01] [Server thread/INFO]: ThreadedAnvilChunkStorage (New Wurld (3)): All chunks are saved
[07:34:01] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_nether
[07:34:01] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[07:34:01] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New Wurld]'/minecraft:the_end
[07:34:01] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[07:34:01] [Server thread/INFO]: ThreadedAnvilChunkStorage (New Wurld (3)): All chunks are saved
[07:34:01] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[07:34:01] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[07:34:01] [main/ERROR]: Setting skin to impaste
[07:34:06] [main/ERROR]: Setting skin to impaste
[07:34:11] [main/INFO]: Stopping!

**Comment by scorbett123** [comment]
Sorry for the slow reply, but please can you also send modified settings, I suspect that the issue may lie in a setting that you have enabled.

**Comment by ZacSharp** [comment]
This happened to me as well a while ago. I can't reproduce it anymore because the schematic (a whole lot of digging) is now built further, but here's the situation I had and the cause I suspect:
I have a schematic that makes it mine a sphere with radius 128 somewhere about 300 blocks away from where I start building with buildInLayers eabled top down and layerHeight 2. Since it is far away it does a full scan and walks to some block towards the -x-z corner of the area. It does not come by any blocks it cares about (yet) because those are all up on the mountain and outside the tickScanRadius. It now just stands at some spot far away from the currently interesting blocks and for whatever reason does not update its goal towards the now loaded and observably wrong blocks.
It did not render goal boxes so I have no idea where it was going (I think it was a JankyGoalComposite).
I think buildInLayers might make a major difference here.

Version is 373ad3ef plus my layerHeight pr (actually the pr is a backport of what I'm using) and a few other customisations not affecting the builder.

**Comment by HackerPhat** [comment]
> Sorry for the slow reply, but please can you also send modified settings, I suspect that the issue may lie in a setting that you have enabled.

allowBreak false
allowInventory true
assumeWalkOnWater true
maxFallHeightNoWater 255
maxFallHeightBucket 255
primaryTimeoutMS 384
planAheadPrimaryTimeoutMS 100
chatControl false
legitMine true

**Comment by ZacSharp** [comment]
No buildInLayers, so my assumption was wrong.
I see you have reduced the timeouts, can you maybe increase them to some absurdly high values (so it doesn't cancel from a timeout) and see whether it finds a path with a bit more time. The problem with the builder and pathing timeouts is that it simply restarts the pathfinder, which fails again unless the world changed.

**Comment by ZacSharp** [comment]
Forget what I said, it's not the timeout either.<details><summary>I managed to reproduce this again and as you can see in the screenshot it even finds a path, but immediately discards it again because it has no throwaway blocks (the reason why it found the path at all is another issue). Since the `PathRenderer` doesn't render `JankyGoalComposite`s I created a selection to show the only goal position and took screenshots until I had one with the path being rendered.</summary>
![2021-11-16_00 40 58](https://user-images.githubusercontent.com/68165024/141870097-93ca1a0e-af2c-47e4-a506-2c7c065cd85c.png)</details> When I manually moved towards the goal until it was inside the `builderTickScanRadius` Baritone did a full rescan (I notice that by the lag it causes because of my massive schematic) and started to mine blocks at the oher end of the schematic. When I did this again, this time with throwaways available, it bridged to the block instead of discarding the path and then continued like when I manually moved it; but since it did not try to continue along the path but instead was about to enter a build scaffolding, break scaffolding loop when it came just about close enough to the "unobserved" block, I think this will not always fix itself like that.
My suspicion is still that it finds the unobservable blocks (out of render distance) and adds them to it's list of wrong blocks. Then it only does local scans because it still has a few wrong blocks on that todo list but if it can't find a path to them, they will never come into range for the local scans so Baritone won't notice they are actually correct (or correct them). Only once the wrong blocks list is empty it will rescan the complete schematic, but since it can't get close enough to remove the last few positions it stalls.

@HackerPhat can you try using `#proc` to see where it is trying to go and then manually move there? If you need to pause it make sure to unpause it at the destination for a short moment so it can do its scanning.
